client/lib/products-values/get-product-billing-term-label.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { TERM_MONTHLY, TERM_ANNUALLY, TERM_BIENNIALLY } from 'calypso/lib/plans/constants';
+import { PRODUCTS_LIST } from 'calypso/lib/products-values/products-list';
+
+/**
+ * Type dependencies
+ */
+import type { ProductSlug } from 'calypso/lib/products-values/types';
+import type { Product } from 'calypso/lib/products-values/products-list';
+
+/**
+ * Returns the billing term label for a product (i.e. "every month", "every year", "every two years").
+ *
+ * @param {ProductSlug} productSlug Product slug
+ * @returns {string} Translated billing term label
```

Comment:
This type is incorrect, which is safe because there's an actual type on this function, but if we're typing it here anyway we might as well note that it can be `undefined`.